Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,12 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+
+using firebase::firestore::remote::Serializer;
+
 namespace {
 
+// Fuzz-test the deserialization process in Firestore. The Serializer reads raw
+// bytes and converts them to a model object.
+void FuzzTestDeserialization(const uint8_t *data, size_t size) {
+  // Blank intentionally.
+}
+
 // Contains the code to be fuzzed. Called by the fuzzing library with
 // different argument values for `data` and `size`.
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
-  // Code to be fuzz-tested here.
+  // Fuzz-test deserialization.
```

Comment:
Nit: this comment is unnecessary because it repeats the function name verbatim. In general, when function/variable/etc. names are clear enough, comments stating what they do are extraneous.

Reply:
Okay. Removed